packages/manager/modules/telecom-spare/src/modems/modems.scss

```diff
@@ -0,0 +1,9 @@
+.telecom-modems-section {
+  @import '~bootstrap4/scss/_functions';
+  @import '~bootstrap4/scss/_variables';
+  @import '~bootstrap4/scss/_mixins';
+  @import '~bootstrap4/scss/_utilities';
+  @import '~bootstrap4/scss/_grid';
+
+  margin: 0 32px;
```

Comment:
Why ? Imo we should try to uniformize our styles and by adding what looks like random `margin` to a component